Package.md

```diff
@@ -0,0 +1,17 @@
+Contains client config utils and Ballerina types for ballerinax connectors.
+
+## Package overview
+The `ballerinax/commons` is a [Ballerina](https://ballerina.io/) library which contains common client config utils and Ballerina types for Ballerinax connectors.
```

Comment:
Is this name `ballerinax/commons` correct?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-client.config/commit/f7d90cfa8f51c55419df68902fb455aeb97978a2